src/containers/PipelineRuns/PipelineRuns.js

```diff
@@ -272,6 +267,92 @@ export /* istanbul ignore next */ class PipelineRuns extends Component {
       </Button>
     );
 
+    const initialHeaders = [
+      {
+        key: 'name',
+        header: intl.formatMessage({
+          id: 'dashboard.tableHeader.name',
+          defaultMessage: 'Name'
+        })
+      },
+      {
+        key: 'pipeline',
+        header: intl.formatMessage({
+          id: 'dashboard.tableHeader.pipeline',
+          defaultMessage: 'Pipeline'
+        })
+      },
+      {
+        key: 'namespace',
+        header: intl.formatMessage({
+          id: 'dashboard.tableHeader.namespace',
+          defaultMessage: 'Namespace'
+        })
+      },
+      {
+        key: 'status',
+        header: intl.formatMessage({
+          id: 'dashboard.tableHeader.status',
+          defaultMessage: 'Status'
+        })
+      },
+      {
+        key: 'transitionTime',
+        header: intl.formatMessage({
+          id: 'dashboard.tableHeader.transitionTime',
+          defaultMessage: 'Last Transition Time'
+        })
+      },
+      {
+        key: 'dropdown',
+        header: ''
+      }
+    ];
+
+    const pipelineRunsFormatted = pipelineRuns.map(template => ({
```

Comment:
would `template` be better named `pipelineRun`?

Reply:
Yep good point - I've updated this for all the files to be more appropriate names